components/dm/command/destroy.go

```diff
@@ -13,24 +13,12 @@
 
 package command
 
-/*
 import (
-	"errors"
-	"os"
-
-	"github.com/fatih/color"
-	"github.com/joomcode/errorx"
-	perrs "github.com/pingcap/errors"
-	"github.com/pingcap/tiup/pkg/cliutil"
-	"github.com/pingcap/tiup/pkg/cluster/meta"
 	operator "github.com/pingcap/tiup/pkg/cluster/operation"
-	"github.com/pingcap/tiup/pkg/cluster/task"
-	"github.com/pingcap/tiup/pkg/logger"
-	"github.com/pingcap/tiup/pkg/logger/log"
-	tiuputils "github.com/pingcap/tiup/pkg/utils"
 	"github.com/spf13/cobra"
 )
 
+// TODO support retain data like cluster?
```

Comment:
I am not sure is there the same requirement.

Reply:
that's why I  left "?", but it will easy to support it, just add the flash like cluster.